README.md

```diff
@@ -211,6 +211,23 @@ By default the module will try to read the credentials from the following locati
 * /etc/sphere-project-credentials
 * /etc/sphere-project-credentials.json
 
+Example usage:
+```js
+import { SphereClient } from 'sphere-node-sdk'
+import { ProjectCredentialsConfig } from 'sphere-node-utils'
+
+const PROJECT_KEY = 'your-project-key'
+
+credentialsConfig = new ProjectCredentialsConfig()
+credentialsConfig._init()
```

Comment:
Yes, we use `enrichCredentials` in order to allow argument passing via command line, but also via config file.